src/shared/components/add-pet-to-compare/add-pet-to-compare.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import style from './add-pet-to-compare.module.scss';
+import { addComparisonPet } from '../../../store/comparison/actions/comparison-handler.action';
+
+import { AddPetToCompareProps } from './add-pet-to-compare.interface';
+import { ComparisonState } from './add-pet-to-compare.interface';
+
+export const AddPetToCompare: React.FC<AddPetToCompareProps> = props => {
+  const dispatch = useDispatch();
+
+  const idsToCompare: string[] = useSelector((state: ComparisonState) => state.comparisonHandler.idsToCompare);
+
+  const findId = () => {
+    return idsToCompare.indexOf(props.id) !== -1 ? true : false;
+  };
```

Comment:
Is there way to improve this function? Either rename it to `hasId` or return id. `findId` should return a value, though.

Reply:
You right
i'll correct it